classes/plugins/ThemePlugin.inc.php

```diff
@@ -33,31 +33,23 @@ function register($category, $path) {
 
 		$request = $this->getRequest();
 		if ($result && $this->getEnabled() && !defined('SESSION_DISABLE_INIT')) {
-			HookRegistry::register('PageHandler::displayCss', array($this, '_displayCssCallback'));
 			$templateManager = TemplateManager::getManager($request);
+			HookRegistry::register('PageHandler::displayCss', array($this, '_displayCssCallback'));
+
+			// Add the stylesheet.
 			$dispatcher = $request->getDispatcher();
 			$templateManager->addStyleSheet($dispatcher->url($request, ROUTE_COMPONENT, null, 'page.PageHandler', 'css', null, array('name' => $this->getName())), STYLE_SEQUENCE_LATE);
+
+			// If this theme uses templates, ensure they're given priority.
+			array_unshift(
+				$templateManager->template_dir,
+				$path = Core::getBaseDir() . DIRECTORY_SEPARATOR . $this->getPluginPath() . '/templates'
```

Comment:
Moot point because / is fine, but why use DIRECTORY _SEPARATOR in one instance and '/templates' in the other?